frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -105,6 +110,18 @@ export const getResourceList = (namespace: string, resList?: any) => {
       namespace,
       prop: 'statefulSets',
     },
+    {
+      isList: true,
+      kind: referenceForModel(PipelineRunModel),
+      namespace,
+      prop: 'pipelineRuns',
```

Comment:
as these are `crd` based resources, will adding `optional: true` make sense?

Reply:
Shouldn't we be plugging into this in the same manner we've done with knative?